components/jaeger-collector.libsonnet

```diff
@@ -82,13 +82,16 @@ local jaegerAgent = import './jaeger-agent.libsonnet';
       local c =
         container.new($.jaeger.deployment.metadata.name, j.image) +
         container.withArgs([
-          '--badger.directory-key=/var/jaeger/store/keys',
-          '--badger.directory-value=/var/jaeger/store/values',
-          '--badger.ephemeral=false',
+          // TODO: Use again, once we move back to badger, currently testing the memory store
+          // '--badger.directory-key=/var/jaeger/store/keys',
+          // '--badger.directory-value=/var/jaeger/store/values',
+          // '--badger.ephemeral=false',
           '--collector.queue-size=4000',
+          '--memory.max-traces 1000',
```

Comment:
The other arguments have `=` between the key and the value. Perhaps it would be good to have it here as well, for consistency's sake?